apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -85,18 +85,17 @@ const SurveyResultsHeader = React.createClass({
 
   componentDidMount() {
     const courses = this.getCoursesForWorkshops();
+    let preselectedWorkshop;
 
     if (courses.length > 0) {
       if (this.props.preselectedWorkshopId) {
-        const preselectedWorkshop = _.find(this.props.workshops, workshop => {
+        preselectedWorkshop = this.props.workshops.find(workshop => {
           return workshop.id === parseInt(this.props.preselectedWorkshopId);
```

Comment:
I just noticed this: make sure you specify the radix 10 in [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt):

"Always specify this parameter to eliminate reader confusion and to guarantee predictable behavior. Different implementations produce different results when a radix is not specified, usually defaulting the value to 10."